tokens/themes/back-office-web.yml

```diff
@@ -0,0 +1,24 @@
+props:
```

Comment:
Should we drop the `web` from the file name?

Reply:
What was your thought process when you first added it?

Reply:
That this is the exact colors we are using for Back Office web and _maybe_ Back Office mobile would have something slightly different so creating that space via indicating its okay to have platform specific sets. Or something like that.

Then my counter thought: does this defeat the purpose of trying to consolidate colors where we have unnecessary deviation? Should we start by trying to use the same sets for all Back Office apps regardless of platform then deviate as needed?

Reply:
And deviation here for Back Office across platforms should be unlikely as these are the high level colors, it would more likely be in the variants.